FAIL: test (test_java.JavaTest)

```
======================================================================
FAIL: test (test_java.JavaTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/smujahid/repos/mozilla/crash-clouseau/tests/test_java.py", line 45, in test
    self.assertEqual(stack, data["frames"])
AssertionError: Lists differ: [{'or[751 chars]me': None, 'module': '', 'changesets': [], 'fu[13792 chars] 71}] != [{'or[751 chars]me': 'mobile/android/base/java/org/mozilla/gec[14251 chars] 71}]

First differing element 3:
{'ori[92 chars]me': None, 'module': '', 'changesets': [], 'fu[75 chars]': 3}
{'ori[92 chars]me': 'mobile/android/base/java/org/mozilla/gec[139 chars]': 3}

  [{'changesets': [],
    'filename': '',
    'function': '',
    'internal': False,
    'line': 0,
    'module': '',
    'node': '',
    'original': 'at '
                'android.text.SpannableStringBuilder.checkRange(SpannableStringBuilder.java:1016)',
    'stackpos': 0},
   {'changesets': [],
    'filename': '',
    'function': '',
    'internal': False,
    'line': 0,
    'module': '',
    'node': '',
    'original': 'at '
                'android.text.SpannableStringBuilder.setSpan(SpannableStringBuilder.java:592)',
    'stackpos': 1},
   {'changesets': [],
    'filename': '',
    'function': '',
    'internal': False,
    'line': 0,
    'module': '',
    'node': '',
    'original': 'at '
                'android.text.SpannableStringBuilder.setSpan(SpannableStringBuilder.java:588)',
    'stackpos': 2},
   {'changesets': [],
-   'filename': None,
+   'filename': 'mobile/android/base/java/org/mozilla/gecko/home/BrowserSearch.java',
    'function': 'format',
    'internal': True,
    'line': 1232,
    'module': '',
    'node': 'tip',
    'original': 'at '
                'org.mozilla.gecko.home.BrowserSearch$9.format(BrowserSearch.java:1232)',
    'stackpos': 3},
   {'changesets': [],
-   'filename': None,
+   'filename': 'mobile/android/base/java/org/mozilla/gecko/home/TwoLinePageRow.java',
    'function': 'update',
    'internal': True,
    'line': 257,
    'module': '',
    'node': 'tip',
    'original': 'at '
                'org.mozilla.gecko.home.TwoLinePageRow.update(TwoLinePageRow.java:257)',
    'stackpos': 4},
   {'changesets': [],
-   'filename': None,
+   'filename': 'mobile/android/base/java/org/mozilla/gecko/home/TwoLinePageRow.java',
    'function': 'updateFromCursor',
    'internal': True,
    'line': 338,
    'module': '',
    'node': 'tip',
    'original': 'at '
                'org.mozilla.gecko.home.TwoLinePageRow.updateFromCursor(TwoLinePageRow.java:338)',
    'stackpos': 5},
   {'changesets': [],
-   'filename': None,
+   'filename': 'mobile/android/base/java/org/mozilla/gecko/home/BrowserSearch.java',
    'function': 'bindView',
    'internal': True,
    'line': 1203,
    'module': '',
    'node': 'tip',
    'original': 'at '
                'org.mozilla.gecko.home.BrowserSearch$SearchAdapter.bindView(BrowserSearch.java:1203)',
    'stackpos': 6},
   {'changesets': [],
-   'filename': None,
+   'filename': 'mobile/android/base/java/org/mozilla/gecko/home/MultiTypeCursorAdapter.java',
    'function': 'getView',
    'internal': True,
    'line': 62,
    'module': '',
    'node': 'tip',
    'original': 'at '
                'org.mozilla.gecko.home.MultiTypeCursorAdapter.getView(MultiTypeCursorAdapter.java:62)',
    'stackpos': 7},
   {'changesets': [],
    'filename': '',
    'function': '',
    'internal': False,
    'line': 0,
    'module': '',
    'node': '',
    'original': 'at android.widget.AbsListView.obtainView(AbsListView.java:2285)',
    'stackpos': 8},
   {'changesets': [],
    'filename': '',
    'function': '',
    'internal': False,
    'line': 0,
    'module': '',
    'node': '',
    'original': 'at android.widget.ListView.makeAndAddView(ListView.java:1790)',
    'stackpos': 9},
   {'changesets': [],
    'filename': '',
    'function': '',
    'internal': False,
    'line': 0,
    'module': '',
    'node': '',
    'original': 'at android.widget.ListView.fillDown(ListView.java:691)',
    'stackpos': 10},
   {'changesets': [],
    'filename': '',
    'function': '',
    'internal': False,
    'line': 0,
    'module': '',
    'node': '',
    'original': 'at android.widget.ListView.fillSpecific(ListView.java:1349)',
    'stackpos': 11},
   {'changesets': [],
    'filename': '',
    'function': '',
    'internal': False,
    'line': 0,
    'module': '',
    'node': '',
    'original': 'at android.widget.ListView.layoutChildren(ListView.java:1608)',
    'stackpos': 12},
   {'changesets': [],
    'filename': '',
    'function': '',
    'internal': False,
    'line': 0,
    'module': '',
    'node': '',
    'original': 'at android.widget.AbsListView.onLayout(AbsListView.java:2132)',
    'stackpos': 13},
   {'changesets': [],
-   'filename': None,
+   'filename': 'mobile/android/base/java/org/mozilla/gecko/widget/themed/ThemedListView.java',
    'function': 'onLayout',
    'internal': True,
    'line': 108,
    'module': '',
    'node': 'tip',
    'original': 'at '
                'org.mozilla.gecko.widget.themed.ThemedListView.onLayout(ThemedListView.java:108)',
    'stackpos': 14},
   {'changesets': [],
    'filename': '',
    'function': '',
    'internal': False,
    'line': 0,
    'module': '',
    'node': '',
    'original': 'at android.view.View.layout(View.java:14858)',
    'stackpos': 15},
   {'changesets': [],
    'filename': '',
    'function': '',
    'internal': False,
    'line': 0,
    'module': '',
    'node': '',
    'original': 'at android.view.ViewGroup.layout(ViewGroup.java:4646)',
    'stackpos': 16},
   {'changesets': [],
    'filename': '',
    'function': '',
    'internal': False,
    'line': 0,
    'module': '',
    'node': '',
    'original': 'at '
                'android.widget.LinearLayout.setChildFrame(LinearLayout.java:1673)',
    'stackpos': 17},
   {'changesets': [],
    'filename': '',
    'function': '',
    'internal': False,
    'line': 0,
    'module': '',
    'node': '',
    'original': 'at '
                'android.widget.LinearLayout.layoutVertical(LinearLayout.java:1527)',
    'stackpos': 18},
   {'changesets': [],
    'filename': '',
    'function': '',
    'internal': False,
    'line': 0,
    'module': '',
    'node': '',
    'original': 'at android.widget.LinearLayout.onLayout(LinearLayout.java:1436)',
    'stackpos': 19},
   {'changesets': [],
    'filename': '',
    'function': '',
    'internal': False,
    'line': 0,
    'module': '',
    'node': '',
    'original': 'at android.view.View.layout(View.java:14858)',
    'stackpos': 20},
   {'changesets': [],
    'filename': '',
    'function': '',
    'internal': False,
    'line': 0,
    'module': '',
    'node': '',
    'original': 'at android.view.ViewGroup.layout(ViewGroup.java:4646)',
    'stackpos': 21},
   {'changesets': [],
    'filename': '',
    'function': '',
    'internal': False,
    'line': 0,
    'module': '',
    'node': '',
    'original': 'at '
                'android.widget.FrameLayout.layoutChildren(FrameLayout.java:453)',
    'stackpos': 22},
   {'changesets': [],
    'filename': '',
    'function': '',
    'internal': False,
    'line': 0,
    'module': '',
    'node': '',
    'original': 'at android.widget.FrameLayout.onLayout(FrameLayout.java:388)',
    'stackpos': 23},
   {'changesets': [],
    'filename': '',
    'function': '',
    'internal': False,
    'line': 0,
    'module': '',
    'node': '',
    'original': 'at android.view.View.layout(View.java:14858)',
    'stackpos': 24},
   {'changesets': [],
    'filename': '',
    'function': '',
    'internal': False,
    'line': 0,
    'module': '',
    'node': '',
    'original': 'at android.view.ViewGroup.layout(ViewGroup.java:4646)',
    'stackpos': 25},
   {'changesets': [],
    'filename': '',
    'function': '',
    'internal': False,
    'line': 0,
    'module': '',
    'node': '',
    'original': 'at '
                'android.widget.RelativeLayout.onLayout(RelativeLayout.java:1055)',
    'stackpos': 26},
   {'changesets': [],
-   'filename': None,
+   'filename': 'mobile/android/base/java/org/mozilla/gecko/GeckoApp.java',
    'function': 'onLayout',
    'internal': True,
    'line': 2338,
    'module': '',
    'node': 'tip',
    'original': 'at '
                'org.mozilla.gecko.GeckoApp$MainLayout.onLayout(GeckoApp.java:2338)',
    'stackpos': 27},
   {'changesets': [],
    'filename': '',
    'function': '',
    'internal': False,
    'line': 0,
    'module': '',
    'node': '',
    'original': 'at android.view.View.layout(View.java:14858)',
    'stackpos': 28},
   {'changesets': [],
    'filename': '',
    'function': '',
    'internal': False,
    'line': 0,
    'module': '',
    'node': '',
    'original': 'at android.view.ViewGroup.layout(ViewGroup.java:4646)',
    'stackpos': 29},
   {'changesets': [],
    'filename': '',
    'function': '',
    'internal': False,
    'line': 0,
    'module': '',
    'node': '',
    'original': 'at '
                'android.widget.RelativeLayout.onLayout(RelativeLayout.java:1055)',
    'stackpos': 30},
   {'changesets': [],
    'filename': '',
    'function': '',
    'internal': False,
    'line': 0,
    'module': '',
    'node': '',
    'original': 'at android.view.View.layout(View.java:14858)',
    'stackpos': 31},
   {'changesets': [],
    'filename': '',
    'function': '',
    'internal': False,
    'line': 0,
    'module': '',
    'node': '',
    'original': 'at android.view.ViewGroup.layout(ViewGroup.java:4646)',
    'stackpos': 32},
   {'changesets': [],
    'filename': '',
    'function': '',
    'internal': False,
    'line': 0,
    'module': '',
    'node': '',
    'original': 'at '
                'android.widget.FrameLayout.layoutChildren(FrameLayout.java:453)',
    'stackpos': 33},
   {'changesets': [],
    'filename': '',
    'function': '',
    'internal': False,
    'line': 0,
    'module': '',
    'node': '',
    'original': 'at android.widget.FrameLayout.onLayout(FrameLayout.java:388)',
    'stackpos': 34},
   {'changesets': [],
    'filename': '',
    'function': '',
    'internal': False,
    'line': 0,
    'module': '',
    'node': '',
    'original': 'at android.view.View.layout(View.java:14858)',
    'stackpos': 35},
   {'changesets': [],
    'filename': '',
    'function': '',
    'internal': False,
    'line': 0,
    'module': '',
    'node': '',
    'original': 'at android.view.ViewGroup.layout(ViewGroup.java:4646)',
    'stackpos': 36},
   {'changesets': [],
    'filename': '',
    'function': '',
    'internal': False,
    'line': 0,
    'module': '',
    'node': '',
    'original': 'at '
                'android.widget.LinearLayout.setChildFrame(LinearLayout.java:1673)',
    'stackpos': 37},
   {'changesets': [],
    'filename': '',
    'function': '',
    'internal': False,
    'line': 0,
    'module': '',
    'node': '',
    'original': 'at '
                'android.widget.LinearLayout.layoutVertical(LinearLayout.java:1527)',
    'stackpos': 38},
   {'changesets': [],
    'filename': '',
    'function': '',
    'internal': False,
    'line': 0,
    'module': '',
    'node': '',
    'original': 'at android.widget.LinearLayout.onLayout(LinearLayout.java:1436)',
    'stackpos': 39},
   {'changesets': [],
    'filename': '',
    'function': '',
    'internal': False,
    'line': 0,
    'module': '',
    'node': '',
    'original': 'at android.view.View.layout(View.java:14858)',
    'stackpos': 40},
   {'changesets': [],
    'filename': '',
    'function': '',
    'internal': False,
    'line': 0,
    'module': '',
    'node': '',
    'original': 'at android.view.ViewGroup.layout(ViewGroup.java:4646)',
    'stackpos': 41},
   {'changesets': [],
    'filename': '',
    'function': '',
    'internal': False,
    'line': 0,
    'module': '',
    'node': '',
    'original': 'at '
                'android.widget.FrameLayout.layoutChildren(FrameLayout.java:453)',
    'stackpos': 42},
   {'changesets': [],
    'filename': '',
    'function': '',
    'internal': False,
    'line': 0,
    'module': '',
    'node': '',
    'original': 'at android.widget.FrameLayout.onLayout(FrameLayout.java:388)',
    'stackpos': 43},
   {'changesets': [],
    'filename': '',
    'function': '',
    'internal': False,
    'line': 0,
    'module': '',
    'node': '',
    'original': 'at android.view.View.layout(View.java:14858)',
    'stackpos': 44},
   {'changesets': [],
    'filename': '',
    'function': '',
    'internal': False,
    'line': 0,
    'module': '',
    'node': '',
    'original': 'at android.view.ViewGroup.layout(ViewGroup.java:4646)',
    'stackpos': 45},
   {'changesets': [],
    'filename': '',
    'function': '',
    'internal': False,
    'line': 0,
    'module': '',
    'node': '',
    'original': 'at '
                'android.widget.LinearLayout.setChildFrame(LinearLayout.java:1673)',
    'stackpos': 46},
   {'changesets': [],
    'filename': '',
    'function': '',
    'internal': False,
    'line': 0,
    'module': '',
    'node': '',
    'original': 'at '
                'android.widget.LinearLayout.layoutVertical(LinearLayout.java:1527)',
    'stackpos': 47},
   {'changesets': [],
    'filename': '',
    'function': '',
    'internal': False,
    'line': 0,
    'module': '',
    'node': '',
    'original': 'at android.widget.LinearLayout.onLayout(LinearLayout.java:1436)',
    'stackpos': 48},
   {'changesets': [],
    'filename': '',
    'function': '',
    'internal': False,
    'line': 0,
    'module': '',
    'node': '',
    'original': 'at android.view.View.layout(View.java:14858)',
    'stackpos': 49},
   {'changesets': [],
    'filename': '',
    'function': '',
    'internal': False,
    'line': 0,
    'module': '',
    'node': '',
    'original': 'at android.view.ViewGroup.layout(ViewGroup.java:4646)',
    'stackpos': 50},
   {'changesets': [],
    'filename': '',
    'function': '',
    'internal': False,
    'line': 0,
    'module': '',
    'node': '',
    'original': 'at '
                'android.widget.FrameLayout.layoutChildren(FrameLayout.java:453)',
    'stackpos': 51},
   {'changesets': [],
    'filename': '',
    'function': '',
    'internal': False,
    'line': 0,
    'module': '',
    'node': '',
    'original': 'at android.widget.FrameLayout.onLayout(FrameLayout.java:388)',
    'stackpos': 52},
   {'changesets': [],
    'filename': '',
    'function': '',
    'internal': False,
    'line': 0,
    'module': '',
    'node': '',
    'original': 'at android.view.View.layout(View.java:14858)',
    'stackpos': 53},
   {'changesets': [],
    'filename': '',
    'function': '',
    'internal': False,
    'line': 0,
    'module': '',
    'node': '',
    'original': 'at android.view.ViewGroup.layout(ViewGroup.java:4646)',
    'stackpos': 54},
   {'changesets': [],
    'filename': '',
    'function': '',
    'internal': False,
    'line': 0,
    'module': '',
    'node': '',
    'original': 'at '
                'android.view.ViewRootImpl.performLayout(ViewRootImpl.java:2035)',
    'stackpos': 55},
   {'changesets': [],
    'filename': '',
    'function': '',
    'internal': False,
    'line': 0,
    'module': '',
    'node': '',
    'original': 'at '
                'android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:1792)',
    'stackpos': 56},
   {'changesets': [],
    'filename': '',
    'function': '',
    'internal': False,
    'line': 0,
    'module': '',
    'node': '',
    'original': 'at '
                'android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:1047)',
    'stackpos': 57},
   {'changesets': [],
    'filename': '',
    'function': '',
    'internal': False,
    'line': 0,
    'module': '',
    'node': '',
    'original': 'at '
                'android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:5896)',
    'stackpos': 58},
   {'changesets': [],
    'filename': '',
    'function': '',
    'internal': False,
    'line': 0,
    'module': '',
    'node': '',
    'original': 'at '
                'android.view.Choreographer$CallbackRecord.run(Choreographer.java:761)',
    'stackpos': 59},
   {'changesets': [],
    'filename': '',
    'function': '',
    'internal': False,
    'line': 0,
    'module': '',
    'node': '',
    'original': 'at '
                'android.view.Choreographer.doCallbacks(Choreographer.java:574)',
    'stackpos': 60},
   {'changesets': [],
    'filename': '',
    'function': '',
    'internal': False,
    'line': 0,
    'module': '',
    'node': '',
    'original': 'at android.view.Choreographer.doFrame(Choreographer.java:544)',
    'stackpos': 61},
   {'changesets': [],
    'filename': '',
    'function': '',
    'internal': False,
    'line': 0,
    'module': '',
    'node': '',
    'original': 'at '
                'android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:747)',
    'stackpos': 62},
   {'changesets': [],
    'filename': '',
    'function': '',
    'internal': False,
    'line': 0,
    'module': '',
    'node': '',
    'original': 'at android.os.Handler.handleCallback(Handler.java:733)',
    'stackpos': 63},
   {'changesets': [],
    'filename': '',
    'function': '',
    'internal': False,
    'line': 0,
    'module': '',
    'node': '',
    'original': 'at android.os.Handler.dispatchMessage(Handler.java:95)',
    'stackpos': 64},
   {'changesets': [],
    'filename': '',
    'function': '',
    'internal': False,
    'line': 0,
    'module': '',
    'node': '',
    'original': 'at android.os.Looper.loop(Looper.java:136)',
    'stackpos': 65},
   {'changesets': [],
    'filename': '',
    'function': '',
    'internal': False,
    'line': 0,
    'module': '',
    'node': '',
    'original': 'at android.app.ActivityThread.main(ActivityThread.java:5105)',
    'stackpos': 66},
   {'changesets': [],
    'filename': '',
    'function': '',
    'internal': False,
    'line': 0,
    'module': '',
    'node': '',
    'original': 'at java.lang.reflect.Method.invokeNative(Native Method)',
    'stackpos': 67},
   {'changesets': [],
    'filename': '',
    'function': '',
    'internal': False,
    'line': 0,
    'module': '',
    'node': '',
    'original': 'at java.lang.reflect.Method.invoke(Method.java:515)',
    'stackpos': 68},
   {'changesets': [],
    'filename': '',
    'function': '',
    'internal': False,
    'line': 0,
    'module': '',
    'node': '',
    'original': 'at '
                'com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:792)',
    'stackpos': 69},
   {'changesets': [],
    'filename': '',
    'function': '',
    'internal': False,
    'line': 0,
    'module': '',
    'node': '',
    'original': 'at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:608)',
    'stackpos': 70},
   {'changesets': [],
    'filename': '',
    'function': '',
    'internal': False,
    'line': 0,
    'module': '',
    'node': '',
    'original': 'at dalvik.system.NativeStart.main(Native Method)',
    'stackpos': 71}]

----------------------------------------------------------------------
Ran 5 tests in 1.490s

FAILED (failures=1)
```

--- a/tests/test_java.py
+++ b/tests/test_java.py
@@ -34,7 +34,17 @@ class JavaTest(unittest.TestCase):
                 return f
 
     def test(self):
-        java_files = java.get_all_java_files(sleep=0.5, retry=10)
+        java_files = [
+            "mobile/android/base/java/org/mozilla/gecko/GeckoApp.java",
+            "mobile/android/base/java/org/mozilla/gecko/BrowserApp.java",
+            "mobile/android/base/java/org/mozilla/gecko/home/BrowserSearch.java",
+            "mobile/android/base/java/org/mozilla/gecko/home/TwoLinePageRow.java",
+            "mobile/android/base/java/org/mozilla/gecko/home/MultiTypeCursorAdapter.java",
+            "mobile/android/base/java/org/mozilla/gecko/widget/themed/ThemedListView.java",
+            "mobile/android/base/java/org/mozilla/gecko/widget/RecyclerViewClickSupport.java",
+            "mobile/android/base/java/org/mozilla/gecko/activitystream/homepanel/StreamRecyclerAdapter.java",
+        ]
+
         for f in self.get_files("./tests/java"):
             data = self.readfile(f)
             stack, files = java.inspect_java_stacktrace(

